checkout-core/src/main/java/com/adyen/checkout/core/log/LogUtil.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2020 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by caiof on 17/12/2020.
+ */
+package com.adyen.checkout.core.log
+
+import android.os.Build
+import com.adyen.checkout.core.exception.NoConstructorException
+
+/**
+ * Utility class with methods related to logs.
+ */
+class LogUtil private constructor() {
+    companion object {
+        private const val CHECKOUT_LOG_PREFIX = "CO."
+        private const val CLASS_NOT_FOUND = "?Unknown?"
+        private const val MAX_TAG_SIZE = 23
+
+        /**
+         * Get the TAG to be used for logging inside Checkout classes.
+         *
+         * @return A String to be used as TAG.
+         */
+        @JvmStatic
+        fun getTag(): String = getTag(CHECKOUT_LOG_PREFIX)
+
+        /**
+         * Get the TAG to be used for logging by the calling class.
+         *
+         * @return A String to be used as TAG with the format "Prefix.ClassName"
+         */
+        // This could be used by merchants if they want to.
+        fun getTag(prefix: String): String {
+            val callerClass = simplifiedCallerClassName
```

Comment:
I was wondering if it's a good idea to use class names. With obfuscation the class names are not very useful, but maybe that is intended 🤷

Reply:
Yeah this is a whole conversation on how to improve our logging structure.